tests/FakeItEasy.Specs/ConfigurationSpecs.cs

```diff
@@ -754,6 +771,11 @@ public sealed override int ReturnSomething()
 
         public class Foo : IFoo
         {
+            public static IFoo BuildFromParamsArray(int[] integers)

```

Comment:
Why "ParamsArray"? There's no `params` modifier

Reply:
Bother. I'd started with a params array and missed this when I converted.